CMakeLists.txt

```diff
@@ -88,7 +88,10 @@ endif()
 # OPENSSL_ROOT_DIR=/usr/local/opt/openssl cmake ..
 include_directories(
     ${GFLAGS_INCLUDE_DIRS} ${GLOG_INCLUDE_DIRS} ${OPENSSL_INCLUDE_DIR}
-    ${Python3_INCLUDE_DIRS})
+)
+if (${Python3_FOUND})
```

Comment:
I think it would be better to make this more explicit, like `option(WITH_PYTHON, ...)`. Otherwise, we just silently don't get python, which could be confusing.